.github/pull_request_template.md

```diff
@@ -10,7 +10,7 @@ Checklists - you can remove one that is not applicable (ie. remove backend check
 If you need help with any of these, please ask :)
 --->
 **Backend checklist**
-- [ ] Formatted my code by running `autoflake --exclude src/proto -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
+- [ ] Formatted my code by running `ruff check --select I --fix . && ruff check . && ruff format .` in `app/backend`
```

Comment:
Also this seems different from the other commands run elsewhere

Reply:
Yes, here I decided that we will always want to do import sorting automatically, the rest is similar. Just that it does not propose the diff - do you think that would be helpful? I find it a bit too verbose for my taste. In a CI pipeline I feel it is better to be more verbose.

Reply:
No this seems good!